src/renderers/webvr/WebVRManager.js

```diff
@@ -14,7 +14,7 @@ function WebVRManager( renderer ) {
 	var device = null;
 	var frameData = null;
 
-	if ( 'VRFrameData' in window ) {
+	if ( typeof window !== 'undefined' && 'VRFrameData' in window ) {
```

Comment:
Would this work?

```js
if ( window && 'VRFrameData' in window ) {
```

Reply:
That won't work if three.js is running in an environment where `window` is not defined, since it would be an undeclared variable. The typeof check returns valid output with undeclared variables, so we can look at the returned string to see if it exists.

Reply:
We are also using the `typeof` check [here](https://github.com/mrdoob/three.js/blob/dev/src/renderers/webgl/WebGLTextures.js#L10). So i think the PR is okay like that.